_data/publications.yaml

```diff
@@ -665,7 +665,7 @@
   date: "2019-12-13"
   author: "Francesco Marchioni"
   affiliation: "Packt / Red Hat"
-  note: "Francesco are the first to publish a book on Quarkus and how to use it for cloud native applications with Java and Quarkus."
+  note: "Francesco is the first to publish a book on Quarkus and how to use it for cloud native applications with Java and Quarkus."
```

Comment:
While you're at it, can you remove the ` with Java and Quarkus` part at the end? It makes no sense.

Reply:
Done